sys/include/random.h

```diff
@@ -22,6 +22,7 @@
  *  - Simple Park-Miller PRNG
  *  - Musl C PRNG
  *  - Fortuna (CS)PRNG
+ *  - Hardware Random Number Generator (non-seedable)
```

Comment:
> I would add it to the documentation about the characteristics of the different PRNG implementations, but I couldn't find those either.

I would just add it here with all the implications @PeterKietzmann mentioned with regards to true randomness and pseudo-randomness.

Reply:
Hm, is there more to add than 'read the data sheet of your MCU' :smiley: ?

Reply:
At least that the HWNRGs vary in how they generate random numbers and that not all are pseudo-number random generators but provide true randomness should also be added as a note.

Reply:
A bit off-topic but anyway: One could spend a note here but IMO the HWRNG implementation should as well indicate its hardware random technology.

Reply:
Spend a note on what @PeterKietzmann?